### DISCUSSION TITLE
无法加载社区主题，请检查你的网络

### DISCUSSION THREAD

**Discussion by bonnytsau** [obsidianzh/forum#25] [Q&A常见问题] [answered]
插件中心也类似，已开科学上网

**Comment by ericaxu** [comment]
请确认这个网址可以加载：

https://raw.githubusercontent.com/obsidianmd/obsidian-releases/master/community-plugins.json

**Reply by bonnytsau** [reply]
确实不能访问，修改window hosts（在：c:/windows/system32/drivers/etc 册立），增加下面的内容，重启obsidian之后就可以加载社区主题和插件了，感谢
```

140.82.113.3 github.com
192.30.253.118 gist.github.com
185.199.109.153 assets-cdn.github.com
199.232.68.133 raw.githubusercontent.com
199.232.68.133 gist.githubusercontent.com
199.232.68.133 cloud.githubusercontent.com
199.232.68.133 camo.githubusercontent.com
199.232.69.194 github.global.ssl.fastly.net
185.199.108.154 github.githubassets.com
185.199.110.153 github.io
```

**Reply by Trevor-Philips** [reply]
谢谢大哥

**Reply by Jayce-Jackson** [reply]
非常感谢

**Reply by Elsieyyy** [reply]
非常感谢！

**Comment by makerjackie** [comment]
更详细的操作指南可以看这里（解决访问github的问题，也就等于解决了obsidian插件/主题的安装问题）
https://github.com/521xueweihan/GitHub520/

**Comment by cyddgh** [comment, marked as answer]
基于521xueweihan/GitHub520/，我转述一份比较低门槛的操作方案 
 https://mp.weixin.qq.com/s/76UNMPzek-bL1wNuabF44w

**Reply by junyi83** [reply]
谢谢~！

**Comment by lillian-who** [comment]
原本想进来说我也遇到了，在家里的电脑上加载不出来插件列表。猜测是因为墙的原因。看到楼主说已科学上网，我就默默吞回去了。学习一下评论区的大神

**Comment by Trevor-Philips** [comment]
您好！
你的邮件已收到！

**Comment by Trevor-Philips** [comment]
您好！
你的邮件已收到！
